release.gradle.kts

```
@@ -0,0 +1,125 @@
+// release.gradle.kts — script plugin (apply from: ...)
+
+apply(plugin = "java")
+apply(plugin = "maven-publish")
+apply(plugin = "signing")
+
+/** -------- Helpers -------- */
+fun projectProperty(k: String, d: String? = null): String? = project.findProperty(k) as String? ?: d
```

the "default" is never actually used (the call sites all use elvis operators for defaults):

```suggestion
fun projectProperty(k: String): String? = project.findProperty(k) as String?
```